clang/test/Driver/sycl-embed-ir-new-offload.cpp

```diff
@@ -0,0 +1,39 @@
+/// Tests for -fsycl-embed-ir
```

Comment:
Not clear to me what these tests are doing in regards to `-fsycl-embed-ir`.  Testing the phases doesn't reflect the actual passing of the option to the `clang-linker-wrapper`.  Using the phases may make more sense for the old model, but as all of the work is being don in the wrapper, the driver test should be simple check to be sure that the new option is being passed to the `clang-linker-wrapper`.

Reply:
Oh, I see you have already added that in the general test below.

Reply:
I was just trying to be 'complete'. I agree..Going forward, I think we can avoid testing for phases

Thanks